presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -279,6 +279,19 @@ public void testArray()
                 .execute(getQueryRunner(), prestoCreateAsSelect("test_array_parameterized_varchar_unicode"));
         arrayVarcharUnicodeDataTypeTest(TestPostgreSqlTypeMapping::postgresArrayDataType)
                 .execute(getQueryRunner(), postgresCreateAndInsert("tpch.test_array_parameterized_varchar_unicode"));
+
+        JdbcSqlExecutor jdbcSqlExecutor = new JdbcSqlExecutor(postgreSqlServer.getJdbcUrl());
+        jdbcSqlExecutor.execute("CREATE TABLE tpch.test_arr(text_arr _text, int_arr _int4)");
```

Comment:
Use `DataTypeTest` with `arrayDataType(integerDataType(), "_int4")`

Reply:
Is there away to use `DataTypeTest` with a predefined table?  
`_int4` can't be created by CTAS from presto.

Reply:
Please see how `TestPostgreSqlTypeMapping#postgresCreateAndInsert` is used. When using this method, the table is created and populated directly in PostgreSQL.

Reply:
Got you. Thanks!